ebms-provider/src/main/resources/db/migrations/V2_payload.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE payload
+(
+    reference_id        VARCHAR(256)		NOT NULL,
+    content_id		    VARCHAR(256)		NOT NULL,
+    content_type		VARCHAR(256)		NOT NULL,
+    payload             TEXT		        NOT NULL,
+    created_date		DATE		        NOT NULL,
```

Comment:
Her kan du bruke _now()_ - funksjonen i Postgres til å generere tidspunktet for deg ala:

created_at  TIMESTAMP DEFAULT now()